pom.xml

```diff
@@ -27,6 +27,7 @@
     <module>events_mgmt</module>
     <module>events_ingress</module>
     <module>events_journal</module>
+    <module>osgi</module>
```

Comment:
Do we want to add this module to the `README.md` as well?

Reply:
yes